### DISCUSSION TITLE
Using 2FA enabled account 'Authentication credentials are missing or invalid' help?

### DISCUSSION THREAD

**Discussion by alouanemed** [fastlane/fastlane#18156] [Q&A]
I m using the latest version and trying to build a relase using math and gym
```

lane :build_release do
    #increment_build_number_cs
	app_store_connect_api_key(
  key_id: "D83848D23",
  issuer_id: "227b0bbf-ada8-458c-9d62-3d8022b7d07f",
  key_content: "-----BEGIN PRIVATE KEY-----
Very private
-----END PRIVATE KEY-----"
)

    puts("Starting release of #{ENV["VERSION_CXS"]}")

    match(app_identifier: "com.x.app",
      type: "appstore")

    gym(
      scheme: "App BETA",
      configuration: "Release",
      output_directory: "build",
      output_name: "app_#{ENV["VERSION_CS"]}.ipa",
      include_bitcode: true,
      clean: true,
      verbose: true
    )
```

The error shows

                    fastlane summary                    |
+------+-----------------------------------+-------------+
| Step | Action                            | Time (in s) |
+------+-----------------------------------+-------------+
| 1    | default_platform                  | 0           |
| 2    | get_version_number_from_xcodeproj | 0           |
| 3    | app_store_connect_api_key         | 0           |
| 💥   | match                             | 3           |
+------+-----------------------------------+-------------+

[22:59:08]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

```

[!] The request could not be completed because:
	Authentication credentials are missing or invalid. - Provide a properly configured and signed bearer token, and make sure that it has not expired. Learn more about Generating Tokens for API Requests https://developer.apple.com/go/?id=api-generating-tokens
```

**Comment by Antol** [comment]
Same for me
